src/vs/workbench/parts/feedback/electron-browser/feedback.ts

```diff
@@ -227,7 +228,14 @@ export class FeedbackDropdown extends Dropdown {
 			})
 			.appendTo($contactUsContainer);
 
-		$('div').append($('a').attr('target', '_blank').attr('href', this.requestFeatureLink).text(nls.localize("request a missing feature", "Request a missing feature")).attr('tabindex', '0'))
+		$('div').append($('a').attr('target', '_blank').attr('href', '#').text(nls.localize("request a missing feature", "Request a missing feature")).attr('tabindex', '0'))
+			.on('click', event => {
+				dom.EventHelper.stop(event);
+				if(!!this.requestFeatureLink){
+					shell.openExternal(this.requestFeatureLink);
```

Comment:
This looks good!

I think it would be simpler to keep the 'href' as `this.requestFeatureLink` above, and then the on click could just have `this.hide()` and let the event bubble.

Reply:
That's a really good point. I'll rework this tomorrow to use that approach.